README.md

```diff
@@ -448,7 +448,7 @@ Here is a collection of research papers about **Monte Carlo Tree Search**.
 - [File an issue](https://github.com/opendilab/LightZero/issues/new/choose) on Github
 - Contact our email (opendilab@pjlab.org.cn)
 
-- We appreciate all the feedbacks and contributions to improve LightZero, both algorithms and system designs. 
+- We appreciate all the feedback and contributions to improve LightZero, both algorithms and system designs. 
```

Comment:
Thank you for your careful proofreading. Any other suggestions for improving LightZero are welcome. Best wishes!